script/deploy.sh

```
@@ -0,0 +1,116 @@
+    "$PROXY_BORROW_ARGS_FILE"
+)
+
+for required_file in "${required_files[@]}"; do
+    if [[ ! -f "$required_file" ]]; then
+        echo "Required file not found: $required_file" >&2
+        exit 1
+    fi
+done
+
+# derived values
+MARKET_ID="$MARKET_NAME.$REGISTRY_ID"
```

Shellcheck (0.11.0)

[warning] MARKET_ID appears unused. Verify use (or export if used externally).

(SC2034)